pydatastructs/graphs/__init__.py

```diff
@@ -11,3 +11,9 @@
     AdjacencyList
 )
 __all__.extend(adjacency_list.__all__)
+
+from . import adjacency_matrix
+from .adjacency_matrix import (
+    AdjacencyMatrix
+)
+__all__.extend(adjacency_matrix.__all__)
```

Comment:
IMO, `AdjacencyMatrix` and `AdjacencyList` shouldn't be there in the global namespace. Please remove them from here and make suitable changes to the import statements.

Reply:
@czgdp1807 but `AdjacencyList` was there from starting should I remove that also

Reply:
Yes.